holidays/countries/singapore.py

```diff
@@ -81,30 +81,32 @@ def __init__(
         See parameters and usage in :py:class:`HolidayBase`.
         """
 
-        self.cnls = _ChineseLuniSolar()
```

Comment:
Agree, and that's why I wrote Singapore that way, but for some reason this assignment didn't monkey patch into Holidays. Since (1) it appeared only once and (2) I wrote the code for Singapore, I took the liberty of changing it for now. 

Will put figuring out a way to monkey patch in the todo.